crates/telio-proxy/src/proxy.rs

```diff
@@ -319,36 +359,60 @@ impl StateEgress {
                 Self::sleep_forever().await;
             }
             ErrorType::SocketIOError => {
-                let socket = match SocketPool::new_udp(
-                    SocketAddr::from((Ipv4Addr::LOCALHOST, 0)),
-                    Some(UdpParams(SocketBufSizes {
-                        rx_buf_size: Some(SOCK_BUF_SZ),
-                        tx_buf_size: Some(SOCK_BUF_SZ),
-                    })),
-                )
-                .await
-                {
-                    Ok(skt) => skt,
-                    Err(err) => {
-                        telio_log_error!("Cannot open socket. {}", err);
-                        return;
+                if let Some(pk) = pk {
+                    if self.peer_socket_backoffs.get_mut(&pk).is_none() {
+                        #[allow(clippy::expect_used)]
+                        let backoff = ExponentialBackoff::new(ExponentialBackoffBounds {
+                            initial: Duration::from_secs(1),
+                            maximal: Some(Duration::from_secs(30)),
```

Comment:
is it neccery to insert / remvoe backoffs ? it feels a bit wastefull.

It should be enough to create on configure, and clear on successful send.